js/base-tool-animator.js

```diff
@@ -1177,6 +1177,7 @@ function baseToolAnimator () {
 	d20plus.anim.animatorTool = {
 		name: "Token Animator",
 		desc: "Manage token animations",
+		mode: true,
```

Comment:
`mode: true` is horribly undescriptive. Use e.g. `mode: "base"` instead, please

Reply:
Fixed.